docs/digger/installation/run-installer.asciidoc

```diff
@@ -0,0 +1,57 @@
+== Run the Digger Installer
+
+To run the Digger installer, you should:
+
+. Step 1
++
+Clone the https://github.com/aerogear/digger-installer[Digger Installer repo]
+
+. Step 2
++
+Update the `inventory-sample` file in the repo to provide the information about your cluster.
+
+. Step 3
++
+Update the configurations of various tasks. You should check the following configurations:
++
+* Check the variables defined in `vars/buildfarm.yml` file and change them if required. 
```

Comment:
As discussed, I will add more variables to the `inventory-sample` file, and change the document here to ask user to make a copy of the that sample inventory file for their own environments.